code/obj/item/fabdisk.dm

```diff
@@ -0,0 +1,29 @@
+obj/item/fabdisk
+	name = "ManuDisk: Empty"
+	desc = "A cartridge for data storage that can be inserted and removed from manufacturers to temporarily add recipes to a manufacturer."
+	icon = 'icons/obj/items/fabdisk.dmi'
+	icon_state = "fabdiskwhi"
+	var/list/datum/manufacture/diskstored = list()
+	var/list/datum/manufacture/disktemp = list()
```

Comment:
What is `temp`? I can probably go work out what this does, but indicates it could possibly use a more descriptive variable name.

Reply:
Temp is where the string of recipes are stored before they get converted to instances, renaming it for clarity would be good.

Reply:
Renamed to temp_recipe_string, bit literal and long but should be fine.